applications/matter_bridge/overlay-bt_max_connections_app_thread_nrf5340.conf

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Set 10 BLE connections, as it is an upper limit supported caused by using thread and BLE.
```

Comment:
Please fix thread->Thread and wifi-> Wi-Fi in all places. The thread using lowercase reminds the thread in the operating system, while we mean Thread protocol name.
```suggestion
# Set 10 BLE connections, as it is an upper limit supported caused by using Thread and BLE.
```